build.sh

```diff
@@ -76,10 +78,11 @@ install_rpms() {
     sed -e "s/^.*default_ccache_name/#    default_ccache_name/g" -i /etc/krb5.conf
 
     # Open up permissions on /boot/efi files so we can copy them
-    # for our ISO installer image
-    find /boot/efi -type f -print0 | xargs -r -0 chmod +r
-    find /boot/efi -type d -print0 | xargs -r -0 chmod +rx
-
+    # for our ISO installer image, not present on s390x
+    if [ "$arch" != "s390x" ]; then
```

Comment:
Could also do e.g. `if [ -d /boot/efi ]; then` to keep it generic.

Reply:
This would also solve it on ppc64le, so I agree, though maybe add a comment in that case stating that `/boot/efi` is not expected to exist on ppc64le and s390x

Reply:
It is better, but I have not hit it on ppc64le, which is surprising in the hindsight.

Reply:
IMO actually -e should be better fit here.

Reply:
Fixed.